src/desktops/dskHostGame.cpp

```diff
@@ -673,10 +675,8 @@ void dskHostGame::CI_Countdown(int countdown)
  */
 void dskHostGame::CI_CancelCountdown()
 {
-    if (isSinglePlayer_)
-    {
+    if (serverType == ServerType::LOCAL)
         return;

```

Comment:
Only reasonable method would be `IsSinglePlayer` to show the intention better. But not really required IMO because the comparison of `serverType` is already clear enough.

Compare: `if(serverType == ServerType::LOBBY)` vs `if(IsServerTypeLobby())` --> Not much gained imo